src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -557,7 +559,14 @@ public List<TranscriptionStatus> getTranscriptionStatuses() {
     @Override
     @Transactional
     public GetTranscriptionByIdResponse getTranscription(Integer transcriptionId) {
-        TranscriptionEntity transcription = transcriptionRepository.findById(transcriptionId).filter(TranscriptionEntity::getIsCurrent)
+        var userIsSuperAdmin = this.userIdentity.userHasGlobalAccess(Set.of(SUPER_ADMIN));
+        TranscriptionEntity transcription = transcriptionRepository.findById(transcriptionId)
+            .filter(TranscriptionEntity::getIsCurrent)
+            // Only SUPER_ADMIN users are allowed to view transcription requests with hidden documents
+            .filter(transcriptionEntity -> userIsSuperAdmin
+                || transcriptionEntity.getTranscriptionDocumentEntities().isEmpty()
+                || transcriptionEntity.getTranscriptionDocumentEntities().stream().noneMatch(TranscriptionDocumentEntity::isHidden)
```

Comment:
Needs integration tests

Reply:
Unit tests added [here](src/test/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImplTest.java). I don't see any benefit in adding integration tests, they are costly to run.